slo-monitor/Makefile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 PACKAGE = k8s.io/perf-tests/slo-monitor
-TAG = 0.15.6
+TAG = 0.15.7
```

Comment:
(Can't comment on files you didn't change) Can you update the go.mod file under slo-monitor from `go 1.20` to 1.21? IIUC when building, Louhi will automatically use the latest patch version of go 1.21.

Reply:
I just checked the Louhi execution logs on previous build. It still build using 1.21.3 with specifying `1.20` in go.mod. I assume, we just need to update the tag version to trigger another build. wdyt?

Reply:
yep the version in go.mod indicates the minimum go version required. Yeah, let's just update the tag and re-trigger louhi build.

Reply:
Removed irrelevant code. PTAL.